planning/behavior_velocity_planner/include/scene_module/scene_module_interface.hpp

```diff
@@ -163,6 +194,19 @@ class SceneModuleManagerInterface
   virtual std::function<bool(const std::shared_ptr<SceneModuleInterface> &)>
   getModuleExpiredFunction(const autoware_auto_planning_msgs::msg::PathWithLaneId & path) = 0;
 
+  virtual bool getActivation([[maybe_unused]] const UUID & uuid) { return false; }
+
+  virtual void updateRTCStatus(
+    [[maybe_unused]] const UUID & uuid, [[maybe_unused]] const bool safe,
+    [[maybe_unused]] const double distance, [[maybe_unused]] const Time & stamp)
+  {
+    return;
+  }
+
+  virtual void removeRTCStatus([[maybe_unused]] const UUID & uuid) { return; }
+
+  virtual void publishRTCStatus([[maybe_unused]] const Time & stamp) { return; }
+
```

Comment:
It is better to implement it in here and remove the following codes in each scene module. 
![image](https://user-images.githubusercontent.com/8327598/170906296-adedb115-03b2-4103-8e8a-6d4e0a0aa372.png)